platform/darwin/test/MGLVersionNumber.m

```diff
@@ -0,0 +1,15 @@
+#import <Mapbox/Mapbox.h>
+
+#import <XCTest/XCTest.h>
+
+@interface MGLVersionTests : XCTestCase
+
+@end
+
+@implementation MGLVersionTests
+
+- (void)testVersionNumber {
+    XCTAssertEqual(1, MapboxVersionNumber);
+}
```

Comment:
Note that `MapboxVersionString` is also available to Objective-C (but not to Swift, owing to its type). I assume it’s also covered by `VERSION_INFO_PREFIX`.